File: CatPredioRsDTO.ts
export interface CatPredioRsDTO {
    preId: number;
    preCodigoCatastral: string;
    preCodigoAnterior?: string;
    preDireccionPrincipal?: string;
    preAreaTotalTer?: number;
    preAreaTotalConst?: number;
    nombrePredio?: string; // Campo computado 
}

File: CatPredioRsDTO.java
package ec.com.intelectus.system.sgb.serviciosbasicos.dtos.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.io.Serializable;
import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class CatPredioRsDTO implements Serializable {
    private Long preId;
    private String preCodigoCatastral;
    private String preCodigoAnterior;
    private String preDireccionPrincipal;
    private BigDecimal preAreaTotalTer;
    private BigDecimal preAreaTotalConst;
    private String nombrePredio; // Campo computado o auxiliar
}